src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -1793,12 +1793,9 @@ tr.status-warning td {
     font-weight: bold;
     padding: 0 1px;
 }
-.log-filter .tab-navigation .tab-control input {
+.log-filter .tab-navigation input[type="radio"] {
     display: none;
```

Comment:
This `display: none` breaks keyboard interaction as the radio input cannot be focused.

Instead, it should be hidden visually only (and focus-visible styles should be added to change the label):

```css
.tab-navigation {
    position: relative;
}

.tab-navigation input[type="radio"] {
  position: absolute;
  pointer-events: none;
  opacity: 0;
}

.tab-navigation input[type="radio"]:checked + .tab-control {
    // Active styles of the tab-control
}

.tab-navigation input[type="radio"]:focus-visible + .tab-control {
    // focus-visible styles of the tab-control
}
```

Reply:
Fixed too!